updating/troubleshooting_updates/gathering-data-cluster-update.adoc

```diff
@@ -21,6 +21,8 @@ include::modules/gathering-clusterversion-history-console.adoc[leveloffset=+2]
 
 include::modules/gathering-clusterversion-history-cli.adoc[leveloffset=+2]
 
+include::modules/update-upgrading-oc-adm-upgrade-status.adoc[leveloffset=+1]
```

Comment:
nit: Maybe move this before the `history` sections, but after the (short) log-data/must-gather section?  I'd certainly recommend folks run this command before recommending they check in with cluster history, in terms of what's most likely to help them understand why their update is having trouble.